vendor-extra/JatsContentBundle/src/ViewConverter/FrontSubjectGroupContentHeaderVisitor.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\JatsContentBundle\ViewConverter;
+
+use DOMNodeList;
+use FluentDOM\DOM\Document;
+use FluentDOM\DOM\Element;
+use Libero\ViewsBundle\Views\ConvertsChildren;
+use Libero\ViewsBundle\Views\SimplifiedVisitor;
+use Libero\ViewsBundle\Views\View;
+use Libero\ViewsBundle\Views\ViewConverter;
+use Libero\ViewsBundle\Views\ViewConverterVisitor;
+use function implode;
+
+final class FrontSubjectGroupContentHeaderVisitor implements ViewConverterVisitor
+{
+    use ConvertsChildren;
+    use SimplifiedVisitor;
+
+    private $converter;
+
+    public function __construct(ViewConverter $converter)
+    {
+        $this->converter = $converter;
+    }
+
+    protected function doVisit(Element $object, View $view, array &$context = []) : View
+    {
+        /** @var Document $document */
+        $document = $object->ownerDocument;
+        $xpath = $document->xpath();
+        $xpath->registerNamespace('jats', 'http://jats.nlm.nih.gov');
+
+        // @todo - add support for other ways of expressing a subject group in JATS.
+        $groups = $xpath->evaluate(implode('/', [
+            'jats:article-meta/jats:article-categories',
+            'jats:subj-group[@subj-group-type = "heading"]/jats:subject',
+        ]), $object);
+
+        if (!$groups instanceof DOMNodeList || $groups->count() === 0) {
+            return $view;
+        }
+
+        $items = [];
+        foreach ($groups as $group) {
+            $items[] = [
+                'content' => [
+                    'text' => $this->convertChildren($group, $context),
```

Comment:
While we are just having text for now, we should use the `link` pattern which would allow for someone to add a `href`, like https://github.com/libero/browser/blob/fddd0f99cbf4aeb3447e1b73b04a441e46991c49/vendor-extra/JatsContentBundle/src/ViewConverter/KeywordGroupTagListVisitor.php#L61-L74